content/sites/ignore-assets.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Ignoring Subsets of Static Assets
+alwaysopen: true
+weight: 5
+---
+
+Worker Sites require [Wrangler](https://github.com/cloudflare/wrangler) - make sure to be on the [latest version](/quickstart/#updating-the-cli) - and the Workers [Unlimited plan](https://workers.cloudflare.com/sites#plans).
```

Comment:
Workers Sites

Reply:
?

Reply:
not Worker Sites

Reply:
oh wait we should fix this for all documentation pages then, they all say `Worker Sites`